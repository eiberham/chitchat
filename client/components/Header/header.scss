.header {
  grid-area: head;
  grid-column: 1 / -1;
  background: rgba(#626882, .8);
  font-size: 3.5rem;
  letter-spacing: 5px;
  color: #fbe5e7;
  p {
    padding: 0;
    margin: 0;
    margin-inline-start: 8rem;
    font-family: inherit;
    font-size: 0.9rem;
    text-transform: uppercase;
    letter-spacing: normal;
  }
}