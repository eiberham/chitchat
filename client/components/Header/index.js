import React from 'react';
import './header.scss';
import { FaRocketchat } from 'react-icons/fa';

const Header = () => {
    return (
        <div className="header">
           <FaRocketchat color="white"/>
           <span> ChitChat</span>
           <p>Spread ur stupidity and never get banned ...</p>
        </div>
    )
};

export default Header;