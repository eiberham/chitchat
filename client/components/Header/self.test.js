import React from 'react';
import { render } from 'enzyme';
import Header from './index';

describe('Header', () => {
    it('Matches snapshot', () => {
        const component = render(<Header />);
        expect(component).toMatchSnapshot();
    });
});