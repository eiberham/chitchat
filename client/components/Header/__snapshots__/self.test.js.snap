// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Matches snapshot 1`] = `
initialize {
  "0": Object {
    "attribs": Object {
      "class": "header",
    },
    "children": Array [
      Object {
        "attribs": Object {
          "color": "white",
          "fill": "currentColor",
          "height": "1em",
          "stroke": "currentColor",
          "stroke-width": "0",
          "style": "color:white",
          "viewBox": "0 0 512 512",
          "width": "1em",
          "xmlns": "http://www.w3.org/2000/svg",
        },
        "children": Array [
          Object {
            "attribs": Object {
              "d": "M256 32C114.6 32 0 125.1 0 240c0 49.6 21.4 95 57 130.7C44.5 421.1 2.7 466 2.2 466.5c-2.2 2.3-2.8 5.7-1.5 8.7S4.8 480 8 480c66.3 0 116-31.8 140.6-51.4 32.7 12.3 69 19.4 107.4 19.4 141.4 0 256-93.1 256-208S397.4 32 256 32zM128 272c-17.7 0-32-14.3-32-32s14.3-32 32-32 32 14.3 32 32-14.3 32-32 32zm128 0c-17.7 0-32-14.3-32-32s14.3-32 32-32 32 14.3 32 32-14.3 32-32 32zm128 0c-17.7 0-32-14.3-32-32s14.3-32 32-32 32 14.3 32 32-14.3 32-32 32z",
            },
            "children": Array [],
            "name": "path",
            "namespace": "http://www.w3.org/2000/svg",
            "next": null,
            "parent": [Circular],
            "prev": null,
            "type": "tag",
            "x-attribsNamespace": Object {
              "d": undefined,
            },
            "x-attribsPrefix": Object {
              "d": undefined,
            },
          },
        ],
        "name": "svg",
        "namespace": "http://www.w3.org/2000/svg",
        "next": Object {
          "attribs": Object {},
          "children": Array [
            Object {
              "data": " ChitChat",
              "next": null,
              "parent": [Circular],
              "prev": null,
              "type": "text",
            },
          ],
          "name": "span",
          "namespace": "http://www.w3.org/1999/xhtml",
          "next": Object {
            "attribs": Object {},
            "children": Array [
              Object {
                "data": "Connect with people from all over the globe ...",
                "next": null,
                "parent": [Circular],
                "prev": null,
                "type": "text",
              },
            ],
            "name": "p",
            "namespace": "http://www.w3.org/1999/xhtml",
            "next": null,
            "parent": [Circular],
            "prev": [Circular],
            "type": "tag",
            "x-attribsNamespace": Object {},
            "x-attribsPrefix": Object {},
          },
          "parent": [Circular],
          "prev": [Circular],
          "type": "tag",
          "x-attribsNamespace": Object {},
          "x-attribsPrefix": Object {},
        },
        "parent": [Circular],
        "prev": null,
        "type": "tag",
        "x-attribsNamespace": Object {
          "color": undefined,
          "fill": undefined,
          "height": undefined,
          "stroke": undefined,
          "stroke-width": undefined,
          "style": undefined,
          "viewBox": undefined,
          "width": undefined,
          "xmlns": "http://www.w3.org/2000/xmlns/",
        },
        "x-attribsPrefix": Object {
          "color": undefined,
          "fill": undefined,
          "height": undefined,
          "stroke": undefined,
          "stroke-width": undefined,
          "style": undefined,
          "viewBox": undefined,
          "width": undefined,
          "xmlns": "",
        },
      },
      Object {
        "attribs": Object {},
        "children": Array [
          Object {
            "data": " ChitChat",
            "next": null,
            "parent": [Circular],
            "prev": null,
            "type": "text",
          },
        ],
        "name": "span",
        "namespace": "http://www.w3.org/1999/xhtml",
        "next": Object {
          "attribs": Object {},
          "children": Array [
            Object {
              "data": "Connect with people from all over the globe ...",
              "next": null,
              "parent": [Circular],
              "prev": null,
              "type": "text",
            },
          ],
          "name": "p",
          "namespace": "http://www.w3.org/1999/xhtml",
          "next": null,
          "parent": [Circular],
          "prev": [Circular],
          "type": "tag",
          "x-attribsNamespace": Object {},
          "x-attribsPrefix": Object {},
        },
        "parent": [Circular],
        "prev": Object {
          "attribs": Object {
            "color": "white",
            "fill": "currentColor",
            "height": "1em",
            "stroke": "currentColor",
            "stroke-width": "0",
            "style": "color:white",
            "viewBox": "0 0 512 512",
            "width": "1em",
            "xmlns": "http://www.w3.org/2000/svg",
          },
          "children": Array [
            Object {
              "attribs": Object {
                "d": "M256 32C114.6 32 0 125.1 0 240c0 49.6 21.4 95 57 130.7C44.5 421.1 2.7 466 2.2 466.5c-2.2 2.3-2.8 5.7-1.5 8.7S4.8 480 8 480c66.3 0 116-31.8 140.6-51.4 32.7 12.3 69 19.4 107.4 19.4 141.4 0 256-93.1 256-208S397.4 32 256 32zM128 272c-17.7 0-32-14.3-32-32s14.3-32 32-32 32 14.3 32 32-14.3 32-32 32zm128 0c-17.7 0-32-14.3-32-32s14.3-32 32-32 32 14.3 32 32-14.3 32-32 32zm128 0c-17.7 0-32-14.3-32-32s14.3-32 32-32 32 14.3 32 32-14.3 32-32 32z",
              },
              "children": Array [],
              "name": "path",
              "namespace": "http://www.w3.org/2000/svg",
              "next": null,
              "parent": [Circular],
              "prev": null,
              "type": "tag",
              "x-attribsNamespace": Object {
                "d": undefined,
              },
              "x-attribsPrefix": Object {
                "d": undefined,
              },
            },
          ],
          "name": "svg",
          "namespace": "http://www.w3.org/2000/svg",
          "next": [Circular],
          "parent": [Circular],
          "prev": null,
          "type": "tag",
          "x-attribsNamespace": Object {
            "color": undefined,
            "fill": undefined,
            "height": undefined,
            "stroke": undefined,
            "stroke-width": undefined,
            "style": undefined,
            "viewBox": undefined,
            "width": undefined,
            "xmlns": "http://www.w3.org/2000/xmlns/",
          },
          "x-attribsPrefix": Object {
            "color": undefined,
            "fill": undefined,
            "height": undefined,
            "stroke": undefined,
            "stroke-width": undefined,
            "style": undefined,
            "viewBox": undefined,
            "width": undefined,
            "xmlns": "",
          },
        },
        "type": "tag",
        "x-attribsNamespace": Object {},
        "x-attribsPrefix": Object {},
      },
      Object {
        "attribs": Object {},
        "children": Array [
          Object {
            "data": "Connect with people from all over the globe ...",
            "next": null,
            "parent": [Circular],
            "prev": null,
            "type": "text",
          },
        ],
        "name": "p",
        "namespace": "http://www.w3.org/1999/xhtml",
        "next": null,
        "parent": [Circular],
        "prev": Object {
          "attribs": Object {},
          "children": Array [
            Object {
              "data": " ChitChat",
              "next": null,
              "parent": [Circular],
              "prev": null,
              "type": "text",
            },
          ],
          "name": "span",
          "namespace": "http://www.w3.org/1999/xhtml",
          "next": [Circular],
          "parent": [Circular],
          "prev": Object {
            "attribs": Object {
              "color": "white",
              "fill": "currentColor",
              "height": "1em",
              "stroke": "currentColor",
              "stroke-width": "0",
              "style": "color:white",
              "viewBox": "0 0 512 512",
              "width": "1em",
              "xmlns": "http://www.w3.org/2000/svg",
            },
            "children": Array [
              Object {
                "attribs": Object {
                  "d": "M256 32C114.6 32 0 125.1 0 240c0 49.6 21.4 95 57 130.7C44.5 421.1 2.7 466 2.2 466.5c-2.2 2.3-2.8 5.7-1.5 8.7S4.8 480 8 480c66.3 0 116-31.8 140.6-51.4 32.7 12.3 69 19.4 107.4 19.4 141.4 0 256-93.1 256-208S397.4 32 256 32zM128 272c-17.7 0-32-14.3-32-32s14.3-32 32-32 32 14.3 32 32-14.3 32-32 32zm128 0c-17.7 0-32-14.3-32-32s14.3-32 32-32 32 14.3 32 32-14.3 32-32 32zm128 0c-17.7 0-32-14.3-32-32s14.3-32 32-32 32 14.3 32 32-14.3 32-32 32z",
                },
                "children": Array [],
                "name": "path",
                "namespace": "http://www.w3.org/2000/svg",
                "next": null,
                "parent": [Circular],
                "prev": null,
                "type": "tag",
                "x-attribsNamespace": Object {
                  "d": undefined,
                },
                "x-attribsPrefix": Object {
                  "d": undefined,
                },
              },
            ],
            "name": "svg",
            "namespace": "http://www.w3.org/2000/svg",
            "next": [Circular],
            "parent": [Circular],
            "prev": null,
            "type": "tag",
            "x-attribsNamespace": Object {
              "color": undefined,
              "fill": undefined,
              "height": undefined,
              "stroke": undefined,
              "stroke-width": undefined,
              "style": undefined,
              "viewBox": undefined,
              "width": undefined,
              "xmlns": "http://www.w3.org/2000/xmlns/",
            },
            "x-attribsPrefix": Object {
              "color": undefined,
              "fill": undefined,
              "height": undefined,
              "stroke": undefined,
              "stroke-width": undefined,
              "style": undefined,
              "viewBox": undefined,
              "width": undefined,
              "xmlns": "",
            },
          },
          "type": "tag",
          "x-attribsNamespace": Object {},
          "x-attribsPrefix": Object {},
        },
        "type": "tag",
        "x-attribsNamespace": Object {},
        "x-attribsPrefix": Object {},
      },
    ],
    "name": "div",
    "namespace": "http://www.w3.org/1999/xhtml",
    "next": null,
    "parent": null,
    "prev": null,
    "root": Object {
      "attribs": Object {},
      "children": Array [
        [Circular],
      ],
      "name": "root",
      "namespace": "http://www.w3.org/1999/xhtml",
      "next": null,
      "parent": null,
      "prev": null,
      "type": "root",
      "x-attribsNamespace": Object {},
      "x-attribsPrefix": Object {},
    },
    "type": "tag",
    "x-attribsNamespace": Object {
      "class": undefined,
    },
    "x-attribsPrefix": Object {
      "class": undefined,
    },
  },
  "_root": [Circular],
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "normalizeWhitespace": false,
    "withDomLvl1": true,
    "xml": false,
  },
}
`;
